src/commands/codepush/patch.ts

```diff
@@ -0,0 +1,126 @@
+import { AppCommand, CommandArgs, CommandResult, help, failure, ErrorCodes, success, getCurrentApp, shortName, longName, required, hasArg, position, name } from "../../util/commandline";
+import { out } from "../../util/interaction";
+import { DefaultApp } from "../../util/profile";
```

Comment:
Unused `DefaultApp` import here, I believe you should remove all unused imports if any.